packages/math-input/src/components/input/mathquill-types.ts

```diff
@@ -30,3 +33,60 @@ export type MathFieldUpdaterCallback = (
     mathField: MathFieldInterface,
     key: Key,
 ) => void;
+
+/**
+ * The MathQuill API (see mathuill.d.ts) does not include types
+ * for cursor() and controller(), and adding these types there
+ * in the MathQuill repo causes unexpected conflicts with other types.
+ *
+ * We don't want to use the cursor and controller default type `any`
+ * so we declare the types here.
+ *
+ * Note: This is different from the MathFieldCursor defined above.
```

Comment:
It should actually be replacing MathFieldCursor, an alias for "any" used to access Cursor. Can you delete that?